test/hotspot/jtreg/vmTestbase/nsk/share/jdi/JDIBase.java

```diff
@@ -150,6 +152,36 @@ protected final void getEventSet() throws JDITestRuntimeException {
         }
     }
 
+    // Special version of getEventSet for ThreadStartEvent/ThreadDeathEvent.
+    // When ThreadStartRequest and/or ThreadDeathRequest are enabled, we can get the events from system threads
+    // unexpected for tests.
+    // The method skips ThreadStartEvent/ThreadDeathEvent events for all threads except the expected one.
```

Comment:
Can you clean up the line lengths so they are close to being even.

Reply:
done